legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/model/DataSpaceExecutableAnalysisInfo.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.generation.analytics.model;
+
+import org.finos.legend.engine.protocol.pure.v1.model.valueSpecification.raw.Lambda;
+
+public class DataSpaceExecutableAnalysisInfo
+{
+    public String id;
+    public String executionContextKey;
+    public String title;
+    public String description;
+    public Lambda query;
+    public String type;
```

Comment:
lets do `PackageableElementPointer` and fill out path with type.

Reply:
I have created a new enum for the type for writing 'inline'

Reply:
just reuse `PackageableElementPointer` no reason to create a new enum. 
if its inline the property will be null

Reply:
so instead of `String type` -> `PackageableElementPointer element`
where we fill out both path and type